### PR TITLE
Fixed temples, added name change function to char sheet, Fixed spinners

### PR DESCRIPTION
**Pull request by ajrb**
Fixed special handling of temple buildings - now can join Akatosh Chantry again by visiting Order of the Hour.
Added name change function to character sheet. Seems to work just fine with save system - just as I thought it would.

### CODE REVIEW

**Comment by ajrb** [comment]
* Fix stat allocation for custom classes to match classic
* Add hold click functionality to spinners

File: Assets/Scripts/Game/UserInterface/UpDownSpinner.cs [hunk 1]
```diff
@@ -121,9 +140,13 @@ public UpDownSpinner(Rect spinnerScreenRect, Rect upButtonRect, Rect downButtonR
             upButton.Position = new Vector2(upButtonRect.xMin, upButtonRect.yMin);
             upButton.Size = new Vector2(upButtonRect.width, upButtonRect.height);
             upButton.OnMouseClick += UpButton_OnMouseClick;
+            upButton.OnMouseDown += UpButton_OnMouseDown;
```
Comment (ajrb) [inline Assets/Scripts/Game/UserInterface/UpDownSpinner.cs]: Why does this class have 2 almost identical constructors Interkarma? If there's no special reason then I could combine the code if you like.

Comment (Interkarma) [reply Assets/Scripts/Game/UserInterface/UpDownSpinner.cs]: Hmm, not sure how that came about. Happy for these to be combined. :)

**Comment by Interkarma** [comment]
Thanks for the fixes!

@numidium It seems @ajrb has made a few of the same spinner fixes as yourself. I'll merge his PR now, as it technically dropped first. Are you OK with resolving the couple of conflicts this will cause in your PR? Let us know if any help needed.

**Comment by numidium** [comment]
@Interkarma Yes, I’m fine with resolving the conflicts. I’ll get on that in a few hours.

**Pull request merged**